docusaurus.config.js

```diff
@@ -240,20 +237,20 @@ const config = {
         toExtensions: ["html"],
         createRedirects(existingPath) {
           return [
-            existingPath.replace('/core', '/develop/advanced'),
-            existingPath.replace('/basics', '/develop/beginner'),
-            existingPath.replace('/intro', '/develop/intro'),
-            existingPath.replace('/architecture', '/build/architecture/'),
-            existingPath.replace('/building-apps', '/build/building-apps'),
+            existingPath.replace("/core", "/develop/advanced"),
+            existingPath.replace("/basics", "/develop/beginner"),
+            existingPath.replace("/intro", "/develop/intro"),
+            existingPath.replace("/architecture", "/build/architecture/"),
+            existingPath.replace("/building-apps", "/build/building-apps"),
             // existingPath.replace('/building-modules', '/build/building-modules'),
-            existingPath.replace('/tooling', '/build/libraries'),
-            existingPath.replace('/migrations', '/build/libraries'),
+            existingPath.replace("/tooling", "/build/libraries"),
+            existingPath.replace("/migrations", "/build/libraries"),
```

Comment:
I haven't introduced that here, this is why I do not fix it, but this looks wrong.

Reply:
This is fine for removal, Ive added the links in (aligning to the previous way) in the other pr open